swan-lake/integration-tools/health-tool.md

```diff
@@ -16,75 +16,129 @@ They are primarily focused on the [Fast Healthcare Interoperability Resources (F
 
 ### Clone the example project
 
 Clone the [artifacts of the example](https://github.com/ballerina-guides/healthcare-samples/tree/main/working-with-health-tool/package-generation) and extract them to a preferred location.
 
 The cloned directory includes the artifacts below that will be required to try out this example (Ballerina project and the FHIR specification files). 
 
-- The [`Ballerina.toml`](https://github.com/ballerina-guides/healthcare-samples/blob/main/working-with-health-tool/package-generation/carinbb-patient-service/Ballerina.toml) file: specifies the dependency of the package (name and version) that is to be generated. 
-- The [`service.bal`](https://github.com/ballerina-guides/healthcare-samples/blob/main/working-with-health-tool/package-generation/carinbb-patient-service/service.bal) file: includes the import of the generated package together with the business logic/mapping implemented using the generated FHIR resource records.
+- The [`ig-carinbb/definitions`](https://github.com/ballerina-guides/healthcare-samples/tree/main/working-with-health-tool/package-generation/ig-carinbb/definitions) directory: includes the definition files of the FHIR specification.
+- The [`carinbb-patient-service`](https://github.com/ballerina-guides/healthcare-samples/tree/main/working-with-health-tool/package-generation/carinbb-patient-service) directory: includes the Ballerina project containing the artifacts (i.e., the `Ballerina.toml`, `Dependencies.toml`, and `service.bal` files) to be executed.
 
-### Generate the package
+#### Generate the package
 
 Follow the steps below to run the Health tool and create the Ballerina package.
 
-1. Navigate to the cloned `working-with-health-tool` directory.
+1. Navigate to the cloned `working-with-health-tool/package-generation` directory.
 
-2. Run the tool with the [required arguments](#command-options) to generate the package.
+2. Run the tool with the [required arguments](#package-generation-command-options) to generate the package.
 
     >**Note:** This example uses the [definitions files](https://github.com/ballerina-guides/healthcare-samples/tree/main/working-with-health-tool/package-generation/ig-carinbb/definitions) downloaded from the **JSON Definitions ZIP archive** of the [Carin BB implementation guide](https://hl7.org/fhir/us/carin-bb/STU2/downloads.html) to generate the package.
 
     ```
-    $ bal health fhir -m package -o ig_carinbb/gen --org-name healthcare_samples --package-name carinbb_package ig_carinbb/definitions/
+    $ bal health fhir -m package -o ig-carinbb/gen --org-name healthcare_samples --package-name carinbb_package ig-carinbb/definitions/
     Ballerina FHIR package generation completed successfully.
     ```
 
+    The generated folder (i.e., `working-with-health-tool/template-generation/ig-uscore/gen`) will contain the following directory structure.
+
+    ```
+    └── gen
+    └── carinbb_package
```

Comment:
`carinbb-package` should be inside the `gen' directory

Reply:
let's remove the `gen` from here

Reply:
Removed.